site/docs/guidelines/filtering.mdx

```diff
@@ -107,10 +107,13 @@ A select is useful in filtering when:
 *   Reducing data by pre-defined segment of data (E.g. In a report select 'Administrator Report' or by comment sentiment, 'Positive', 'Neutral', 'Negative')
 *   Shows the options available for that field (E.g. Country demographic, 'Australia', 'England', 'United States')
```

Comment:
Grammar nitpick but can we fix up these "e.g's" so they're consistent with the new one you've added? (remove upper case E and parenthesis)

Reply:
Done 👍